client/blocks/import/capture/index.tsx

```diff
@@ -99,6 +99,9 @@ const CaptureStep: React.FunctionComponent< Props > = ( {
 	 */
 	useEffect( recordScanningEvent, [ isAnalyzing ] );
 	useEffect( recordScanningErrorEvent, [ analyzerError ] );
+	useEffect( () => {
+		recordTracksEvent( trackEventName, trackEventParams );
```

Comment:
When I jump in into this line, it was hard for me to understand why we are triggering it. It is not required, but I would suggest following the current patterns, so why not wrap it in a function semantically correct name so that it can be clear why we are calling it.

For example: `useEffect( recordCaptureScreen, [] );`

Reply:
You're right. I should follow the current patterns. Fixed at https://github.com/Automattic/wp-calypso/pull/64696/commits/332bc33af86b6b654e8e8cc86e07ed3553fc934a